<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>real_time</class>
 <widget class="QWidget" name="real_time">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridlayout">
      <item row="2" column="0" colspan="3">
       <widget class="QTextEdit" name="textEdit"/>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_start">
        <property name="text">
         <string>start </string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="maximumSize">
         <size>
          <width>220</width>
          <height>280</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="lineEdit">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="pushButton_add">
        <property name="text">
         <string>add</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="pushButton_clear">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>4</horstretch>
          <verstretch>2</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>5</height>
         </size>
        </property>
        <property name="text">
         <string>clear</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_pause">
        <property name="text">
         <string>pause</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
